docs/ROOT_STORAGE_DAEMONS.md

```diff
@@ -182,7 +180,7 @@ few additional notes for supporting these setups:
   implemented by systemd. Transparently for your client code connecting to the
   socket of your storage daemon will result in the storage to be started. For
   that it is simply necessary to inform systemd about the socket you'd like it
-  to listen on on behalf of your daemon and minimally modify the daemon to
+  to listen on behalf of your daemon and minimally modify the daemon to
   receive the listening socket for its services from systemd instead of
   creating it on its own. Such modifications can be minimal, and are easily
```

Comment:
This part of the doc was difficult to understand and I think could be improved, but I wasn't able to come up with a better wording.